packages/jest-core/src/__tests__/TestScheduler.test.js

```diff
@@ -126,7 +137,7 @@ test('schedule tests run in serial if the runner flags them', async () => {
   await scheduler.scheduleTests(tests, {isInterrupted: jest.fn()});
 
   expect(mockSerialRunner.runTests).toHaveBeenCalled();
-  expect(mockSerialRunner.runTests.mock.calls[0][5].serial).toBeTruthy();
+  expect(mockSerialRunner.runTests.mock.calls[0][2].serial).toBeTruthy();
 });
 
 test('should bail after `n` failures', async () => {
```

Comment:
@SimenB Could you suggest a better implementation for mocking eventEmitters (extended from Emittery)? Need to get the `should bail after 'n' failures` test passing

Reply:
why doesn't the `jest.fn` work?